[live555] build failure

<!-- ⚠ Please use the generated issue template instead to report a port build failure. ⚠ -->
<!-- ⁉ You can find its location at the end of the console output. ⁉ -->

**Host Environment**

- OS: [Windows/Linux]
- Compiler: revision

**To Reproduce**

Steps to reproduce the behavior:

`./vcpkg install live555`

**Failure logs**

- Cut and paste the appropriate build messages from the console output.
- Attach ***every failure log file*** mentioned in the console output.
-
C:\vcpkg>vcpkg install live555 --triplet x64-windows
Computing installation plan...
The following packages will be built and installed:
    live555[core]:x64-windows -> 2022-07-14
Detecting compiler hash for triplet x64-windows...
Restored 0 package(s) from C:\Users\user\AppData\Local\vcpkg\archives in 6.317 ms. Use --debug to see more details.
Installing 1/1 live555:x64-windows...
Building live555[core]:x64-windows...
-- Note: live555 only supports static library linkage. Building static library.
-- Downloading http://www.live555.com/liveMedia/public/live.2022.07.14.tar.gz -> live.2022.07.14.tar.gz...
[DEBUG] To include the environment variables in debug output, pass --debug-env
[DEBUG] Feature flag 'binarycaching' unset
[DEBUG] Feature flag 'manifests' = off
[DEBUG] Feature flag 'compilertracking' unset
[DEBUG] Feature flag 'registries' unset
[DEBUG] Feature flag 'versions' unset
[DEBUG] Downloading http://www.live555.com/liveMedia/public/live.2022.07.14.tar.gz
Error: Failed to download from mirror set:
http://www.live555.com/liveMedia/public/live.2022.07.14.tar.gz: failed: status code 404

[DEBUG] D:\a\_work\1\s\src\vcpkg\base\downloads.cpp(766):
[DEBUG] Time in subprocesses: 0 us
[DEBUG] Time in parsing JSON: 8 us
[DEBUG] Time in JSON reader: 0 us
[DEBUG] Time in filesystem: 1599 us
[DEBUG] Time in loading ports: 0 us
[DEBUG] Exiting after 468.8 ms (425425 us)

CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:32 (message):

      Failed to download file with error: 1
      If you use a proxy, please check your proxy setting. Possible causes are:

      1. You are actually using an HTTP proxy, but setting HTTPS_PROXY variable
         to `https://address:port`. This is not correct, because `https://` prefix
         claims the proxy is an HTTPS proxy, while your proxy (v2ray, shadowsocksr
         , etc..) is an HTTP proxy. Try setting `http://address:port` to both
         HTTP_PROXY and HTTPS_PROXY instead.

      2. You are using Fiddler. Currently a bug (https://github.com/microsoft/vcpkg/issues/17752)
         will set HTTPS_PROXY to `https://fiddler_address:port` which lead to problem 1 above.
         Workaround is open Windows 10 Settings App, and search for Proxy Configuration page,
         Change `http=address:port;https=address:port` to `address`, and fill the port number.

      3. Your proxy's remote server is out of service.

      In future vcpkg releases, if you are using Windows, you no longer need to set
      HTTP(S)_PROXY environment variables. Vcpkg will simply apply Windows IE Proxy
      Settings set by your proxy software. See (https://github.com/microsoft/vcpkg-tool/pull/49)
      and (https://github.com/microsoft/vcpkg-tool/pull/77)

      Otherwise, please submit an issue at https://github.com/Microsoft/vcpkg/issues

Call Stack (most recent call first):
  scripts/cmake/vcpkg_download_distfile.cmake:273 (z_vcpkg_download_distfile_show_proxy_and_fail)
  ports/live555/portfile.cmake:3 (vcpkg_download_distfile)
  scripts/ports.cmake:147 (include)


error: building live555:x64-windows failed with: BUILD_FAILED
error: Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
Then check for known issues at:
    https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+live555
You can submit a new issue at:
    https://github.com/microsoft/vcpkg/issues/new?template=report-package-build-failure.md&title=[live555]+Build+error
Include '[live555] Build error' in your bug report title, the following version information in your bug description, and attach any relevant failure logs from above.
    vcpkg-tool version: 2022-09-20-522aa94e9d261c7d7b2f079bf2591ca62df5c714
    vcpkg-scripts version: 083f103cf 2022-10-09 (8 weeks ago)


Please use the prefilled template from C:\vcpkg\installed\vcpkg\issue_body.md when reporting your issue.

<!-- ‼ **The console output by itself is not enough to investigate a build failure.** ‼ -->

**Additional context**

The live555 developers change the version of product. See http://www.live555.com/liveMedia/public/

 In terms of the portfile.cmake:
--- actually ----
vcpkg_download_distfile(ARCHIVE
    URLS "http://www.live555.com/liveMedia/public/live.**2022.07.14**.tar.gz"
    FILENAME "live.2022.07.14.tar.gz"
    SHA512 382544d9d9fe200699669a1f3301efb4ccec0193499c95b532ea923c380b1ec6fa721a4118d36a447ba9df08575f185498f244293c66bbe97cff0482eab033c7
)
--- but expected (for last live555 update) ---
vcpkg_download_distfile(ARCHIVE
    URLS "http://www.live555.com/liveMedia/public/live.**2022.12.01**.tar.gz"
    FILENAME "live.2022.12.01.tar.gz"
    SHA512 bb5dc80b5b1621e04fb8a100bd3deff190efb757da10e6cfc652d6eaa878f6a3e063b2f2219d5d83d6fb6892b55be55eafe2dd43f42a559e1f931130b45584b1
)

--- a/ports/live555/portfile.cmake
+++ b/ports/live555/portfile.cmake
@@ -6,9 +6,9 @@ vcpkg_download_distfile(ARCHIVE
     SHA512 bb5dc80b5b1621e04fb8a100bd3deff190efb757da10e6cfc652d6eaa878f6a3e063b2f2219d5d83d6fb6892b55be55eafe2dd43f42a559e1f931130b45584b1
 )
 
-vcpkg_extract_source_archive_ex(
-    OUT_SOURCE_PATH SOURCE_PATH
-    ARCHIVE ${ARCHIVE} 
+vcpkg_extract_source_archive(
+    SOURCE_PATH
+    ARCHIVE "${ARCHIVE}" 
     PATCHES
         fix-RTSPClient.patch
 )
@@ -29,6 +29,6 @@ file(GLOB HEADERS
 )
 
 file(COPY ${HEADERS} DESTINATION "${CURRENT_PACKAGES_DIR}/include")
-file(INSTALL "${SOURCE_PATH}/COPYING" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
+vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/COPYING")
 
 vcpkg_copy_pdbs()

--- a/ports/live555/portfile.cmake
+++ b/ports/live555/portfile.cmake
@@ -1,9 +1,9 @@
 vcpkg_check_linkage(ONLY_STATIC_LIBRARY) 
 
 vcpkg_download_distfile(ARCHIVE
-    URLS "http://www.live555.com/liveMedia/public/live.2022.07.14.tar.gz"
-    FILENAME "live.2022.07.14.tar.gz"
-    SHA512 382544d9d9fe200699669a1f3301efb4ccec0193499c95b532ea923c380b1ec6fa721a4118d36a447ba9df08575f185498f244293c66bbe97cff0482eab033c7
+    URLS "http://www.live555.com/liveMedia/public/live.2022.12.01.tar.gz"
+    FILENAME "live.2022.12.01.tar.gz"
+    SHA512 bb5dc80b5b1621e04fb8a100bd3deff190efb757da10e6cfc652d6eaa878f6a3e063b2f2219d5d83d6fb6892b55be55eafe2dd43f42a559e1f931130b45584b1
 )
 
 vcpkg_extract_source_archive_ex(

--- a/ports/live555/vcpkg.json
+++ b/ports/live555/vcpkg.json
@@ -1,6 +1,6 @@
 {
   "name": "live555",
-  "version-date": "2022-07-14",
+  "version-date": "2022-12-01",
   "description": "A complete RTSP server application",
   "homepage": "http://www.live555.com/liveMedia",
   "license": "GPL-3.0-or-later",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -4569,7 +4569,7 @@
       "port-version": 0
     },
     "live555": {
-      "baseline": "2022-07-14",
+      "baseline": "2022-12-01",
       "port-version": 0
     },
     "llfio": {

--- a/versions/l-/live555.json
+++ b/versions/l-/live555.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "3eee4a32779504666e21804b89bc3569f729e043",
+      "git-tree": "7c55bad3936e67fe4e56228533acc22e6bae2572",
       "version-date": "2022-12-01",
       "port-version": 0
     },

--- a/versions/l-/live555.json
+++ b/versions/l-/live555.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "3eee4a32779504666e21804b89bc3569f729e043",
+      "version-date": "2022-12-01",
+      "port-version": 0
+    },
+    {
       "git-tree": "6f99761c9c260c5b654cd1028610b2edaa3036e0",
       "version-date": "2022-07-14",
       "port-version": 0

